Sierra/challenge_14/stations_challenge.py

```diff
@@ -0,0 +1,111 @@
+        self.location = location
+    
+    def show_info(self):
+        print(f'{self.station_name} station is located at {self.location}')
+
+
+class SubwayStation(Station):
+    def __init__(self, lines, station_name, location):
+        super().__init__(station_name, location)
+        self.lines = lines
+
+    def show_info(self):
+        print(f'{self.lines} are the lines that stop at {self.station_name} station. {self.station_name} station is located at {self.location}.')
+
+
+print('Question 2: Make an example subway station')
+'''
+Using your SubwayStation class, instantiate a subway station with the info below. 
+Then run the show_info() method to make sure you get the station_name, location, and lines printed out
+
+station_name: '14th street'
+location: '14th street and 7th avenue'
+lines: ['1', '2', '3', 'L']
+'''
+subway_station1 = SubwayStation(station_name='14th street', location='14th street and 7th avenue', lines=['1', '2', '3', 'L'])
+
+subway_station1.show_info()
+
+print('Question 3: Making the BusStation Class')
+
+'''
+Using the Station class below as the parent, make a child class called BusStation
+
+BusStation should:
+-have an additional attribute called 'routes' that is user-defined as a list during initialization. 
+    this will indicate where buses can go from this station. For example, ['DC', 'Philly', 'Pittsburgh']
+-have an additional attribute called 'open' that is always initalized as True (a boolean variable)
+-have additional methods called open_station() and close_station() to open and close the station
+-override the show_info() method from Station to display the bus routes and if the station is open, in addition to the station name and location
+'''
+class BusStation(Station):
+    def __init__(self, routes, station_name, location):
+        super().__init__(station_name, location)
+        self.routes = routes
+        self.open = True
+    
+    def open_station(self):
+        self.open == True
+        self.open = 'open'
+
+    def close_station(self):
+        self.open == False
+        self.open = 'closed'
+
+    def show_info(self):
+        self.open_station()
+        print(f'The bus routes are {self.routes} and the station is {self.open}. {self.station_name} is located at {self.location}.')
+
+
+print('Question 4: Make an example bus station')
+'''
+Using your BusStation class, instantiate a bus station with the info below. 
+Then, run the show_info() method to make sure you get the station_name, location, routes, and whether the station is open printed out
+Then, demonstrate that you can close and open the bus station
+    i.e. that the show_info() method reflects correctly when the station is open versus closed
+
+station_name: 'NYC Megabus Stop'
+location: '34th street and 12th avenue'
+lines: ['Boston', 'DC', 'Philly']
+'''
+bus_station1 = BusStation(routes=['Boston', 'DC', 'Philly'], station_name='NYC Megabus Stop', location='34th street and 12th avenue')
+
+bus_station1.show_info()
```

Comment:
oops. not a big deal but you forgot to demonstrate that you can close and open the bus station. But it's not a big deal.